homeassistant/components/climate/hive.py

```diff
@@ -134,6 +136,48 @@ def set_temperature(self, **kwargs):
             for entity in self.session.entities:
                 entity.handle_update(self.data_updatesource)
 
+    @property
+    def is_aux_heat_on(self):
+        """Return true if aux heater."""
+        boost_status = None
+        boost_status_return = False
+        if self.device_type == "Heating":
+            boost_status = self.session.heating.get_boost(self.node_id)
+        elif self.device_type == "HotWater":
+            boost_status = self.session.hotwater.get_boost(self.node_id)
+
+        if boost_status == "ON":
+            boost_status_return = True
+
+        return boost_status_return

```

Comment:
```py
return boost_status == "ON"
```

Reply:
@MartinHjelmare 

so 'return boost_status == "ON"' will return true if `boost_status == "ON"'? and if the condition is not `=="ON"` then false is returned?

so lines 149 to 152 can be replaced with that one line in your comments, and `boost_status_return` is not needed at all?

Reply:
Yes! :+1:

Reply:
well @MartinHjelmare that is pretty clever

Update made as above